airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/bigquery/test_primary_key_streams/final/airbyte_ctes/test_normalization/nested_stream_with_complex_columns_resulting_into_long_names_partition_ab1.sql

```diff
@@ -8,6 +8,7 @@ select
     _airbyte_nested_stream_with_complex_columns_resulting_into_long_names_hashid,
     json_extract_array(`partition`, "$['double_array_data']") as double_array_data,
     json_extract_array(`partition`, "$['DATA']") as DATA,
+    json_extract_array(`partition`, "$['column___with__quotes']") as column___with__quotes,
```

Comment:
due to PR https://github.com/airbytehq/airbyte/pull/5026, the raw json blob also contains some sanitized columns that we can extract from where quote characters have already been replaced by `_`